convert2rhel/unit_tests/actions/pre_ponr_changes/backup_system_test.py

```diff
@@ -141,6 +141,7 @@ def test_backup_redhat_release_error_os_release_file(self, backup_redhat_release
             (
                 """     missing   d /usr/share/info/ed.info.gz
                 S.5.?..T. c     /etc/yum.repos.d/CentOS-Linux-AppStream.repo
+                S.5.?..T. c     /etc/yum.repos.d/CentOS-Linux-AppStream.repo
                 S.5.?..T.        /etc/yum.repos.d/CentOS-Linux-AppStream.repo
```

Comment:
@r0x0d I don't know what the different statuses mean really. But what does c indicate here? We have it listed 3 times and we do AFAIK expect to have this correct:

```
                    {
                        "status": "S5T",
                        "file_type": "c",
                        "path": "/etc/yum.repos.d/CentOS-Linux-AppStream.repo",
                    },
                    {
                        "status": "S5T",
                        "file_type": None,
                        "path": "/etc/yum.repos.d/CentOS-Linux-AppStream.repo",
                    },
```

Reply:
http://ftp.rpm.org/max-rpm/s1-rpm-verify-output.html

> c appears only if the file is a configuration file.

Reply:
Better resource: https://www.man7.org/linux/man-pages/man8/rpm.8.html

>        The format of the output is a string of 9 characters, a possible
>        attribute marker:
> 
>               a %artifact a build side-effect file (such as buildid links)
>               c %config configuration file.
>               d %doc documentation file.
>               g %ghost file (i.e. the file contents are not included in the package payload).
>               l %license license file.
>               m %missingok file missing is not a verify failure.
>               r %readme readme file.
> 
>        from the package header, followed by the file name.  Each of the
>        9 characters denotes the result of a comparison of attribute(s)
>        of the file to the value of those attribute(s) recorded in the
>        database.  A single "." (period) means the test passed, while a
>        single "?" (question mark) indicates the test could not be per‐
>        formed (e.g. file permissions prevent reading).  Otherwise, the
>        (mnemonically emBoldened) character denotes failure of the corre‐
>        sponding --verify test:
> 
>               S file Size differs
>               M Mode differs (includes permissions and file type)
>               5 digest (formerly MD5 sum) differs
>               D Device major/minor number mismatch
>               L readLink(2) path mismatch
>               U User ownership differs
>               G Group ownership differs
>               T mTime differs
>               P caPabilities differ

Reply:
Because you're adding another entry to the `rpm_va_output` test parameter, you need to add also an entry to the `expected` parameter.

```
                    {
                        "status": "S5T",
                        "file_type": "c",
                        "path": "/etc/yum.repos.d/CentOS-Linux-AppStream.repo",
                    },
                    {
                        "status": "S5T",
                        "file_type": "c",
                        "path": "/etc/yum.repos.d/CentOS-Linux-AppStream.repo",
                    },
                    {
                        "status": "S5T",
                        "file_type": None,
                        "path": "/etc/yum.repos.d/CentOS-Linux-AppStream.repo",
                    },
```

Reply:
Right, but since we back it up only once we just want it listed once right?

Reply:
Ah, right, indeed.